tests/integration/test_end_to_end_dcp.py

```diff
@@ -105,16 +105,19 @@ class TestSmartSeq2Run(TestEndToEndDCP):
 
     def test_smartseq2_run(self):
         runner = self.ingest_store_and_analyze_dataset(dataset_fixture='Smart-seq2')
-        try:
-            self.assertEqual(1, len(runner.primary_bundle_uuids))
-            self.assertEqual(1, len(runner.secondary_bundle_uuids))
-            expected_files = self.expected_results_bundle_files(runner.primary_bundle_uuids[0],
-                                                                self.SS2_ANALYSIS_OUTPUT_FILES_REGEXES)
-            results_bundle_manifest = self.data_store.bundle_manifest(runner.secondary_bundle_uuids[0])
+        with Timeout(2 * 3600) as to:
```

Comment:
We should set this to 1 hour and 50 mins even if gitlab times out at 2 hours. This will account for any gitlab specific spin up prior to this test running

Reply:
fixed